RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -210,12 +210,18 @@ boolean auto_doPhoneQuest()
 boolean auto_haveMonkeyPaw()
 {
 	static item paw = $item[cursed monkey\'s paw];
-	return auto_is_valid(paw) && (item_amount(paw) > 0 || have_equipped(paw));
+	return auto_is_valid(paw) && possessEquipment(paw);
+}
+
+boolean auto_canMonkeyPawWish()
+{
+	static item paw = $item[cursed monkey\'s paw];
+	return (auto_is_valid(paw) || in_glover()) && possessEquipment(paw);
 }
```

Comment:
We can wish in G-Lover but can't use the paw for anything else like banishes or olfacts.